services/scheduleService.js

```diff
@@ -0,0 +1,57 @@
+const schedule = require('node-schedule'),
+  pinModel = require('../models/pinModel'),
+  ipfsAPI = require('ipfs-api'),
+  _ = require('lodash'),
+  bunyan = require('bunyan'),
+  config = require('../config'),
+  Promise = require('bluebird'),
+  log = bunyan.createLogger({name: 'plugins.ipfs.scheduleService'});
+
+/**
+ * @module scheduleService
+ * @description ping ipfs by specified time in config
+ * @see {@link ../config.json}
+ */
+
+module.exports = (network) => {
```

Comment:
please, replace promises with async/await - and don't forget to check your code with integration tests